### PR TITLE
Add AEMM mock interruption documentation

### PR DESCRIPTION
**Pull request by haugenj**
Issue #, if available:
#208 

Description of changes:
Moves the information about using AEMM to simulate a spot interruption from this issue into a more permanent location

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/aemm_interruption_testing.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+for the local tests that use a kind cluster, and [here](https://github.com/aws/aws-node-termination-handler/blob/master/test/eks-cluster-test/run-test#L117)
+for the eks-cluster e2e tests.
+
+Check out the [ReadMe](https://github.com/aws/aws-node-termination-handler/tree/master/test) in our test folder for more
+info on the e2e tests. 
+
+---
+
+\* Only the first two unique IPs to request data from AEMM receive spot itn information in the response. This was introduced
+in AEMM v1.6.0. This can be overridden with a configuration paramter. For previous versions there is no unique IP restriction.
```
Comment (brycahta) [inline docs/aemm_interruption_testing.md:90]: paramter --> parameter

**Review by brycahta — CHANGES_REQUESTED** [review]

File: docs/aemm_interruption_testing.md (hunk 1 above)
Comment (haugenj) [reply docs/aemm_interruption_testing.md:90]: f

Comment (brycahta) [reply docs/aemm_interruption_testing.md:90]: was about to ask how spellcheck didn't catch it

Comment (bwagner5) [reply docs/aemm_interruption_testing.md:90]: 🤖

**Review by brycahta — APPROVED** [review]
Looks Good To Send ✈️

**Review by bwagner5 — APPROVED** [review]
nice!

**Pull request merged**